domain/schema_validator.go

```diff
@@ -80,17 +76,14 @@ func (s *schemaValidator) Stop() {
 	defer s.mux.Unlock()
 	s.isStarted = false
 	s.latestSchemaVer = 0
-	s.detalItemInfos = nil
-	s.itemSchemaVers = nil
+	s.deltaSchemaInfos = make([]deltaSchemaInfo, 0, maxNumberOfDiffsToLoad)
```

Comment:
Because of the implement of `enqueue`, maybe we can set the capability value as `maxNumberOfDiffsToLoad +1`